src/services/User Management/server.js

```diff
@@ -0,0 +1,15 @@
+const express = require("express");
+const consume = require("./rmq/rmqConsumerServer");
+const connectToDB = require("./prisma/connectToDB");
+const app = express();
+require("dotenv").config();
```

Comment:
This should be the first or second line in the file so that if the database and RabbitMQ need to get anything from the `.env` file they will not be able to do this because this file is not configured yet.